trillian_admin_api.proto

```diff
@@ -21,6 +21,7 @@ option java_outer_classname = "TrillianApiProto";
 package trillian;
 
 import "trillian.proto";
+import "github.com/google/trillian/crypto/keyspb/keyspb.proto";
```

Comment:
import "crypto/keyspb/keyspb.proto", so it's consistent with other internal imports?